sympy/stats/crv.py

```diff
@@ -138,12 +139,7 @@ def set(self):
                 "Set of Conditional Domain not Implemented")
 
 
-class ContinuousDistribution(Basic):
```

Comment:
are you removing the distinction between continuous and discrete distributions?

Reply:
The difference is still present in other classes such as `SingleContinuousDistribution`, but `ContinuousDistribution` and `DiscreteDistribution` classes were identical, `__call__` being the only method described. Adding `handle_compound_dist` would also lead to some more code repitition.